test/relayserver/ServerTestEnvironment.ts

```diff
@@ -74,7 +75,7 @@ abiDecoder.addABI(TestVerifierEverythingAccepted.abi);
 // @ts-ignore
 abiDecoder.addABI(TestDeployVerifierEverythingAccepted.abi);
 
-export const LocalhostOne = 'http://localhost:8095';
+export const LocalhostOne = `${RIF_RELAY_HOST}:${RIF_RELAY_PORT}`;
```

Comment:
maybe also this constant can be removed to use the one declared in `test/Utils.ts` or if you prefer, we could move all the constants here, or alternatively, we could define a `test/constants.ts` file